### ISSUE TITLE
Vertical mode ?

### ISSUE THREAD

**Issue by anorgeot** [Laproxi/DeSmuME-NX#4]
Would it be possible to have a vertical mode option for games that use only with the touch screen or for play in detached mode ?

**Comment by Laproxi** [comment]
I am not sure what you mean: could you rephrase/rewrite that?

**Comment by anorgeot** [comment]
Sorry for my english,  here is an example:
![vertical-switch](https://user-images.githubusercontent.com/18189276/42840566-98805da4-8a07-11e8-85f2-562b383e8866.jpeg)
